src/main/res/values-de/strings.xml

```diff
@@ -261,7 +261,7 @@ schwarze Liste gesetzt wird</string>
 
         Wollen Sie fortfahren?]]></string>
   <string name="sign_in_warning_dialog_title">Anmeldungswarnung</string>
-  <string name="password_warning_dialog_title">Warnung</string>
+  <string name="password_warning_dialog_title">Warnung ⚠️</string>
```

Comment:
```suggestion
  <string name="password_warning_dialog_title">Warning ⚠️</string>
```

Reply:
This is the German version, it was already translated so why not leave it like that?